src/test/app/wasm_fixtures/all_host_functions/Cargo.toml

```diff
@@ -10,7 +10,7 @@ edition = "2024"
 crate-type = ["cdylib"]
 
 [dependencies]
-xrpl-std = { git = "https://github.com/ripple/craft.git", branch = "main", package = "xrpl-std" }
+xrpl-std = { git = "https://github.com/ripple/craft.git", branch = "last-hfs2", package = "xrpl-std" }
```

Comment:
Is the `last-hfs2` temporary? I also see `lastdevnet` was used in `src/test/app/wasm_fixtures/all_keylets/Cargo.toml`. Will we change back to main later?

Reply:
Yes and yes. The changes aren't available in `main`/`lastdevnet` yet. Thanks for the reminder, I'll open that craft PR in a minute.

Reply:
https://github.com/ripple/craft/pull/158